about.html

```diff
@@ -0,0 +1,131 @@
+          <li class="mobile-nav"><a href="#">Sponsor</a></li>
+          <li class="mobile-nav"><a href="#">News</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="home">
+      <h3>About...</h3>
+      <img src="assets/aboutLOGO.png" alt="program logo" id="about-logo" />
+      <p id="p1b">
+        ‘The Experience’ is a free, all-night Gospel music concert featuring
+        some of the most celebrated and exceptional psalmists from Nigeria and
+        around the world. This phenomenal concert is convened annually by Pastor
+        Paul Adefarasin, the Metropolitan of all House On The Rock, Lagos,
+        Nigeria.
+        <br /><br />
+        It is the perfect atmosphere for hundreds of thousands of people to
+        gather in one accord to praise and worship God with unfettered
+        expressions from dusk till dawn irrespective of class, race or
+        denomination.
+      </p>
+      <p id="p2b">
+        For further information,<br />kindly email us at
+        <span>teproject@houseontherock.org.ng</span><br />HOUSE ON THE ROCK
+      </p>
+    </section>
+    <section class="bottom">
+      <div class="divider">
+        <h1>over<span>500,000</span>Voices</h1>
+      </div>
+      <div class="sum-head">
+        <h2>House on the Rocks The Experience 2022</h2>
+        <hr>
+        <p>The logo of the 2022 Experience Concert was agreed on unanimously, as it is the 16th edition</p>
+        <img src="assets/aboutLOGO.png" alt="program logo" id="sum-logo" />
+      </div>
+      <hr id="linnn">
+      <div class="sum-body">
+        <h2>See the past Experience Edition</h2>
+        <hr>
+        <p>Take a look at the last two The Experience Concerts which  took place at the same venue</p>
+        <div class="cont">
+          <div class="pic-p1">
+            <h3>2020</h3>
+            <p>The Experience 14</p>
+          </div>
+          <div class="pic-p2">
+            <h3>2021</h3>
+            <p>The Experience 15</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-top" id="mobile1">
+        <h3>Partners</h3>
+        <hr />
+        <div class="sponsors">
+          <img src="./assets/sp1.jpg" alt="sponsor1" />
+          <img src="./assets/sp2.jpg" alt="sponsor2" />
+          <img src="./assets/sp3.png" alt="sponsor3" />
+          <img src="./assets/sp4.png" alt="sponsor4" />
+        </div>
+      </div>
+      <div class="footer-bottom" id="mobile2">
+        <h2>shyn designs</h2>
+        <div class="footer-bottoma">
+          <p>2022 Microverse Capstone Project.</p>
+          <p>Some Rights Reserved.</p>
+        </div>
+      </div>
+      <div class="desktop-only" id="desktop1">
+        <div class="footer-bottom">
+          <h2 class="wit">shyn designs</h2>
+          <div class="footer-bottoma" >
+            <p class="wit">2022 Microverse Capstone Project.</p>
+            <p class="wit">Some Rights Reserved.</p>
+          </div>
```

Comment:
Great job for noticing that the footer here is dark 💯 
- [ ] Kindly remove the empty white space at the bottom of the footer 👍 
- [ ] Kindly update the content of the footer, it should be the logo on the left and the text on the right and they're kind of centered in the container 👍 (this is for both pages)

![image](https://user-images.githubusercontent.com/50721479/204088039-9198d3e2-0973-4e5b-b8c1-7455f814db77.png)

- [ ] For mobile version, great job on making it light colored but the content is not positioned correctly and the font and text size used is also not like in the guidelines 😓 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/204088264-a30ec027-5afe-4ccb-8ffa-3cff780023f3.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/204088279-fbaf7064-74c3-4c8e-9aeb-b8c3b0bc7766.png)

Reply:
- [ ] This issue is not  well resolved but in my opinion, I'd mark it optional as I see your effort in making it close to the design template :x: 


- As your content are not bold enough, there are still ways to fix this. 
This is just a tip you may choose to try.
You can remove this line of code 

```
.footer-bottom {
    padding-left: 10em;
}
```

then add 
```
justify-content: space-evenly`, 
```
to this `.footer-bottom`

```
.footer-bottom {
    display: flex;
    padding: 1em;
    align-items: center;
}

```

 as this is better compared to your current design below:


![Screenshot from 2022-11-26 17-44-26](https://user-images.githubusercontent.com/59397159/204099628-9e62b6be-9b98-445b-8e41-d7225b8b3a86.png)